wormhole-connect/src/components/DemoAppHeader.tsx

```diff
@@ -0,0 +1,67 @@
+import './styles.css';
+import React, { useEffect } from 'react';
+import { useState } from 'react';
+import { WormholeConnectConfig } from 'config/types';
+
+interface Props {
+  onCustomConfigChange: (config: WormholeConnectConfig | undefined) => void;
+}
+
+const LOCAL_STORAGE_KEY = 'wormhole-connect:demo:custom-config';
+
+export default function DemoAppHeader(props: Props) {
+  let [customConfigOpen, setCustomConfigOpen] = useState(false);
+  let [customConfigInput, setCustomConfigInput] = useState(
+    localStorage.getItem(LOCAL_STORAGE_KEY) ?? '',
+  );
+
+  const updateCustomConfig = (e: any) => {
+    const input = e.target.value;
+    setCustomConfigInput(input);
+  };
+
+  const emitCustomConfig = () => {
+    localStorage.setItem(LOCAL_STORAGE_KEY, customConfigInput);
```

Comment:
would be nice if also we can load custom config form a query param

Reply:
oh cool idea. let me see if I can do that

Reply:
done. great idea man